spec/services/hyrax/collection_types/create_service_spec.rb

```diff
@@ -22,4 +22,23 @@
       end.to change(Hyrax::CollectionTypeParticipant, :count).by(described_class::DEFAULT_OPTIONS[:participants].count)
     end
   end
+
+  describe '.add_default_participants' do
+    let(:coltype) { create(:collection_type) }
+
+    it 'adds the default participants to a collection type' do
+      expect(Hyrax::CollectionTypeParticipant).to receive(:create!).exactly(2).times
+      described_class.add_default_participants(coltype.id)
+    end
+  end
+
+  describe '.add_participants' do
+    let(:participants) { [{ agent_type: Hyrax::CollectionTypeParticipant::GROUP_TYPE, agent_id: 'test_group', access: Hyrax::CollectionTypeParticipant::MANAGE_ACCESS }] }
+    let(:coltype) { create(:collection_type) }
+
+    it 'adds the participants to a collection type' do
+      expect(Hyrax::CollectionTypeParticipant).to receive(:create!)
+      described_class.add_participants(coltype.id, participants)
+    end
```

Comment:
Please add an unhappy path spec for adding participants (e.g. when it fails)

Reply:
Regarding the unhappy path, can you demonstrate what a failed create would look like (and what it's impact would be)

Reply:
Opened issue #1797 to address these concerns.

Reply:
👍